Documentation/file-specifications/camera-calibration.md

```diff
@@ -85,3 +128,140 @@ The camera calibration file is a csv and contains all of the information to load
 840.2,839.2,-6325.8
 0.1,1760,1760
 ```
+
+## Backend Calculations
+
+### Overview
+
+This section describes the calculations that are performed by the backend to convert the camera calibration file into an internal Camera object.
+
+### Image Plane
+
+
+The image plane is the location of the DRR image in world space. The center of the image plane is calculated by the following equation:
+
+```{note}
+The below formula applies to VTKCam 1.0 and MayaCam 2.0. For MayaCam 1.0, `c_x`` and `c_y` are replaced by `u0` and `v0` respectively. The value for `z` is also not calculated and is instead pulled from the camera calibration file.
+
+`DRR_X` and `DRR_Y` are the width and height of the DRR image respectively.
+
+The translation matrix is represented by `t` and the rotation matrix is represented by `R`.
+```
+
+The constant of `-0.5` is used so that the `z` value is the average of the `f_x` and `f_y` values and negated to be consistent with the MayaCam 1.0 file format.
```

Comment:
```suggestion
The constant `-0.5` is used so that the `z` value is the average of the `f_x` and `f_y` values, and it is negated to be consistent with the MayaCam 1.0 file format.
```